deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -347,6 +362,53 @@ serviceMonitors:
     namespaceSelector:
       any: true
 
+  localpv-lvm:
+    enabled: true
+
+    ## Endpoints of the selected service to be monitored
+    endpoints:
+      ## Name of the endpoint's service port
+      ## Mutually exclusive with targetPort
+      #  port: ""
+      port: exporter
+
+      ## HTTP path to scrape for metrics
+      #  path: /metrics
+      path: /metrics
+
+    ## Example
+
+    # selector:
+    #   matchLabels:
+    #     cas-type: localpv-lvm
+
+    selector:
+      matchLabels:
+        openebs.io/cas-type: localpv-lvm
+
```

Comment:
Is this service present by default in the cluster for the lvm volumes?

Reply:
We will have to create service from lvm-localPV and add servicemonitor through here. 
I thought of doing this in this PR, but doing it in the next PR  will be good. Will raise the separate PR for adding service(lvm-localPV) as well as service monitor(openebs-monitoring).